### PR TITLE
Adds work view.

### LINKED ISSUE #33: View a work
Update the view page to look like this as much as possible:

https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/420930436

### PR DESCRIPTION
**Pull request by justinlittman**
closes #33

## Why was this change made?
To allow viewing the details for a work.

## How was this change tested?
Unit, locally.

## Which documentation and/or configurations were updated?
settings.yml

### CODE REVIEW

**Comment by justinlittman** [comment]
image

File: app/models/work.rb [hunk 1]
```diff
@@ -35,4 +35,11 @@ class Work < ApplicationRecord
       transition deposited: :version_draft
     end
   end
+
+  sig { returns(T.nilable(String)) }
+  def purl
+    return nil unless druid
+
+    File.join(Settings.purl_url, T.must(druid).delete_prefix('druid:'))
```
Comment (jcoyne) [inline app/models/work.rb:43]: Can you add a test for this?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/models/work.rb (hunk 1 above)
Comment (mjgiarlo) [reply app/models/work.rb:43]: I'm curious about using `File.join` to do a string/URL join.

Comment (justinlittman) [reply app/models/work.rb:43]: File.join is smart about the trailing slash. (We do the same in Argo: https://github.com/sul-dlss/argo/blob/8c31437630dcce1fdb8f9a9c6fb5f8ff5a8a9a32/app/components/external_links_component.rb#L10)

Comment (justinlittman) [reply app/models/work.rb:43]: If we were running on Windows, this would be problematic but I'm thinking that is unlikely.

Comment (justinlittman) [reply app/models/work.rb:43]: Test added.

**Review by jcoyne — APPROVED** [review]

File: app/models/work.rb (hunk 1 above)
Comment (mjgiarlo) [reply app/models/work.rb:43]: ok, thanks!

**Pull request merged**